File: JS Essentials/2.0 Simple Calculations and Operations/08. Fish Tank.js
function fishTank(arg){
    let [long, width,  hight, percent] = [...arg]
    let size = long * width * hight * 0.001
    let pr = percent * 0.01
    return (size * (1 - pr)).toFixed(3)     
 }
 
// console.log(fishTank([85, 75, 47, 17]))

File: Python Essentials/2.0 Simple Calculations and Operations/08. Fish Tank.py
long, width, height, percent = [float(input()) for _ in range(4)]
size = long * width * height * 0.001
pr = percent * 0.01
print(f'{size * (1 - pr):.3f}')

 
'''
85
75
47
17
#248.689
'''
